docs/immortal-empires/high-elves.md

```diff
@@ -0,0 +1,41 @@
+# High Elves
+
+## Tyrion - Eataine
+
+* **Route I - Defender of Ulthuan**
+  * Prince Tyrion is the Defender of Ulthuan and the protector of its people, a heavy burden entrusted to him by the Phoenix King Finubar. Among the Dark Elves of Naggaroth he is known as the Reaper, to the Goblins of Red-Axe Pass he is Orcbane, and to the north the Norse know him as Mankiller. For two centuries he has stood between the Elves of Ulthuan and their many foes as an unbreakable shield against the darkness.
+  * Destroy factions:
+    * Seducers of Slaanesh, The Dreadfleet, Broken Axe, The Drowned
+  * **Reward**:
+    * _Ulthun Secure_: Income from Ports: +50%, +10 Leadership (factionwide) 
+_Reward flavour text:_ Not since the Age of Colonisation have the Elves known such prosperity.
+icon_ritual_currency_hef_bundle.png
+
+* **Route II - All's Fair in Love and War**
+  * Malekith's ruinous ambitions have brought death and destruction to Ulthuan for centuries. Elf has fought against elf for far too long. The civil war must end.
+  * Destroy factions: 
+    * Naggarond
+  * Tyrion concerns himself only with the twin duties of protecting Ulthuan and its Everqueen, the beloved Alarielle. However, the latter is seen less as a duty than it is a calling of the heart - it is common knowledge that Tyrion is Alarielle's consort, and has been for many years.
+  * Occupy settlement:
+    * Lothern, Gaean Vale
+  * Build Landmark:
+    * Court of the Phoenix King
+  * **Reward**: 
+    * _Dawn of a New Age_: All remaining Dark Elf factions suffer -10 control, Recruitment duration: -1 turn for all units (minimum 1), Recruit rank: +3 for all units
```

Comment:
Remove the recruitement duration and give +10 Leaderhsip when fighting against Dark Elves. The -10 control can be tricky to code, but feasible. Recruit rank: +2 feels good enough for balance.